debezium-core/src/main/java/io/debezium/config/CommonConnectorConfig.java

```diff
@@ -1043,6 +1044,16 @@ public static SnapshotQueryMode parse(String value, String defaultValue) {
             .withDescription(
                     "When 'snapshot.mode' is set as configuration_based, this setting permits to specify whenever the data should be snapshotted or not in case of error.");
 
+    public static final Field LOG_POSITION_CHECK_ENABLED = Field.create("log.position.check.enable")
```

Comment:
```suggestion
    public static final Field LOG_POSITION_CHECK_ENABLED = Field.createInternal("log.position.check.enable")
```

let's keep it undocumented at the start till we evaluate if it is something necessary